packages/flutter/test/rendering/viewport_test.dart

```diff
@@ -1144,4 +1144,45 @@ void main() {
       '   its intrinsic dimensions.\n',
     );
   });
+
+  testWidgets('Handles Infinity - Infinity on iOS', (WidgetTester tester) async {
```

Comment:
Can you document how we end up with infinity - infinity?  It isn't clear to me how that is happening in the test.

Reply:
I've tried to add an explanation. Not sure if it's super clear but it's also touching on areas that are a bit tough to explain - I'm hoping @Hixie or @goderbauer can help fill that out if it's not clear enough